src/images/coblox_bitcoincore.rs

```diff
@@ -197,7 +198,7 @@ impl Image for BitcoinCore {
 
     fn ready_conditions(&self) -> Vec<WaitFor> {
         vec![
-            WaitFor::message_on_stdout("Flushed wallet.dat"),
+            WaitFor::message_on_stdout("bitcoind startup sequence completed."),
```

Comment:
To make the relationship between these constants clearer, can we extract this into a non-pub `const` and use `format!` above to configure the command?